### ISSUE TITLE
[Bug Report][2.5.0] v-combobox @change is firing after v-btn @click

### ISSUE THREAD

**Issue by Agendum** [vuetifyjs/vuetify#13698]
### Environment
**Vuetify Version:** 2.5.0
**Vue Version:** 2.6.11
**Browsers:** Edge 90.0.818.56
**OS:** Windows 10

### Steps to reproduce
1) Go to the JSFiddle
2) It auto-runs, and auto-focuses to start appending text on the initial combo box item
3) Type a letter (i.e. the number 1) to append a letter to the text (i.e. 'a1')
4) Click the red 'CLICK' button

### Expected Behavior
Because I removed focused out of the v-combobox control, I would expect the @change to be immediately fired, as the model would be updated. Then the button's @click would fire and it could read the updated value.

### Actual Behavior
The button's @click fires before the model of the v-combobox's model is updated, and therefore the @change also fires after @click. On the JSFiddle you can see this in the 'Value During Click' text.

### Reproduction Link
<a href="https://jsfiddle.net/CodeVirtue/p8sng47e/" target="_blank">https://jsfiddle.net/CodeVirtue/p8sng47e/</a>

### Other comments
Basically, if I have a v-combobox on view and the user clicks a button which should evaluate the model of the combobox, I have no idea if my code is looking at the fresh value of the v-combobox, or if it will be updated later.

**Comment by KaelWD** [comment]
```js
setTimeout(() => this.valDuringClick = this.val)
```
